src/hypothesis/internal/compat.py

```diff
@@ -478,3 +478,10 @@ def implements_iterator(it):
     # the x flag because it's Python 2, so there are no places where this can
     # be thrown.
     FileExistsError = None
+
+
+if PY2:
```

Comment:
Just checking - I assume you've done a `git grep` for `math.floor` and replaced it throughout the codebase?